src/CMakeLists.txt

```diff
@@ -328,13 +333,20 @@ add_custom_target(hotgym
 ## MNIST Spatial Pooler Example
 #
 set(src_executable_mnistsp mnist_sp)
-add_executable(${src_executable_mnistsp} "examples/mnist/MNIST_SP.cpp")
-target_link_libraries(${src_executable_mnistsp} ${core_library})
+#! set(MNIST_DIR ${mnist-SOURCE_DIR})
+#! find_package(MNIST  REQUIRED)
```

Comment:
this fails! MNIST is not properly linked/installed

@dkeeney you've written the External CMake, can you please help me with the new dependency? external/mnist_data.cmake. 

- I want to download and extract MNIST .gz data, 
- optionally (preferably?) I want to build the repo there, which includes the data + read methods for its format.

Reply:
Ok, I will look at it.